### PR TITLE
Fix Chinese Translations for DME & Update Chinese Translations

### PR DESCRIPTION
**Pull request by zero6six**
I'm so sorry I put the translated language file for DME into the wrong folder by mistake, so I opened this PR to fix it.By the way, I updated translation for AE2, cns(CraftingNetherStar) and QB.

In the process of translation, I found some strange settings (or glitchs?) about `.displayName`:

1. Repeated <thermalexpansion:frame:64>.

https://github.com/Nomi-CEu/Nomi-CEu/blob/98ec44d31833ede7d8625bc2ed0116c30f57c88f/overrides/scripts/ThermalExpansion.zs#L390C1-L397

2. Only define the name of <cns:star_fragment>, ingnoring <cns:star_core>.
https://github.com/Nomi-CEu/Nomi-CEu/blob/98ec44d31833ede7d8625bc2ed0116c30f57c88f/overrides/scripts/extendedcrafting.zs#L281

I wonder if I could make a commit about changing these hardcoded strings to softcode strings?For example, `<thermalexpansion:frame:64>.displayName = "Thermal Machine Casing";` will be  replaced with`<thermalexpansion:frame:64>.displayName = (game.localize("thermal.machineframe"))`,and  I will add `thermal.machineframe=Thermal Machine Casing` to contenttweaker\lang\en_us.lang.

If you accept that, I will refactor all the hardcoded strings about item names next weekend.

### CODE REVIEW

**Comment by IntegerLimit** [comment]
Note that CNS and some other mods will be replaced by Nomi Labs soon?

If you want to, go ahead! That would be very helpful. I am not sure of the timeline when I will have time to move everything to GroovyScript and Nomi-Labs, so you doing it will be very helpful.

*Instead of setting the display name in the scripts, find the original translation key for those items, and replace the value for those (both English and Chinese) in a lang file, in the relevant folder (or a new one), in the resources folder.*

That should probably go in a new Pull Request. I'll merge this first!

File: overrides/resources/appliedenergistics2/lang/zh_cn.lang
Comment (IntegerLimit) [inline overrides/resources/appliedenergistics2/lang/zh_cn.lang:1]: If this file is not needed, please delete it. Else, add it back (not sure if this is accidental)

File: overrides/resources/cns/lang/zh_cn.lang
Comment (IntegerLimit) [inline overrides/resources/cns/lang/zh_cn.lang:1]: Is CNS the actual modid? Can you find the proper mod id for it?

**Review by IntegerLimit — CHANGES_REQUESTED** [review]
Please resolve the two comments below.

File: overrides/resources/cns/lang/zh_cn.lang
Comment (zero6six) [reply overrides/resources/cns/lang/zh_cn.lang:1]: Yes, the following is the mcmod.info information in "CraftingNetherStar 1.12.1 -1.1.jar".
```json
[
{
  "modid": "cns",
  "name": "Crafting Nether Star",
  "description": "Makes nether stars craftable. // Hace que las estrellas del Nether sean crafteables",
  "version": "1.1",
  "mcversion": "1.12.1",
  "url": "https://minecraft.curseforge.com/projects/craftingnetherstar",
  "updateUrl": "",
  "authorList": ["ZeckG"],
  "credits": "Carlos Bravo",
  "logoFile": "cns.png",
  "screenshots": [],
  "dependencies": []
}
]
```

**Review by IntegerLimit — APPROVED** [review]

**Pull request merged**